junit/build.gradle

```diff
@@ -0,0 +1,49 @@
+apply plugin: 'maven-publish'
+apply plugin: 'me.modmuss50.remotesign'
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
+
+archivesBaseName = "fabric-loader-junit"
+version = rootProject.version
+group = rootProject.group
+
+def ENV = System.getenv()
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	api project(":")
+
+	compileOnly(platform("org.junit:junit-bom:5.9.2"))
+	compileOnly("org.junit.platform:junit-platform-launcher")
```

Comment:
compileOnly? Surely that's a runtime dep

Reply:
Yes, good spot left over from testing with the snapshot version. I have changed this to expose the bom and `org.junit.jupiter:junit-jupiter-engine` as api, and the platform-launcher as impl. This makes it so you will just need to depend on `testImplementation "net.fabricmc:fabric-loader-junit:${project.loader_version}"` in your project.